com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/modelinference/ExtendableInferredModelAssociator.java

```diff
@@ -23,7 +23,7 @@
 @Singleton
 public class ExtendableInferredModelAssociator extends InferredModelAssociator {
 
-  @Inject(optional = true)
+  @Inject

```

Comment:
!= null check below is not needed anymore then

Reply:
Sure, done.